static/js/message_events.js

```diff
@@ -259,6 +260,10 @@ export function update_messages(events) {
                 compose_fade.set_focused_recipient("stream");
             }
 
+            if (stream_name) {
```

Comment:
Does it seem reasonable to just skip if there's no stream_name?

Reply:
Yes, though we should add a comment for why this is correct.

Reply:
After thinking a bit more, I think it's more correct to remove this check -- it's quite confusing logic to inspect `stream_name`, which is a proxy for whether `event.stream_id` is known to `sub_store`. Further, the drafts algorithm just cares about stream IDs, not stream names, and should correctly handle being passed a `stream_id` not present in `sub_store` anyway. Deleted this and added a comment.